### PR TITLE
[tf ssd300] get rid of conda environment again

### PR DESCRIPTION
**Pull request by awshaichen**

### CODE REVIEW

**Review by aws-mesharma — COMMENTED** [review]
Can we remove first line of going to install guide and add the step to install tensorflow-model-server-neuron here?

File: src/examples/tensorflow/ssd300_demo/README.md [hunk 1]
```diff
@@ -16,31 +16,23 @@ For this demo, launch one inf1.xlarge EC2 instance. We recommend using the lates
...
 
 ```bash
 sudo apt update
-sudo apt install g++ python3-dev unzip
+sudo apt install g++ python3-dev python3-venv unzip
 sudo apt install tensorflow-model-server-neuron
+python3 -m venv env
+source ./env/bin/activate
 pip install pip setuptools --upgrade
 pip install -r ./requirements.txt --extra-index-url=https://pip.repos.neuron.amazonaws.com
```
Comment (aws-mesharma) [inline src/examples/tensorflow/ssd300_demo/README.md:35]: sudo apt-get install tensorflow-model-server-neuron
git clone https://github.com/aws/aws-neuron-sdk.git
cd aws-neuron-sdk/src/examples/tensorflow/ssd300_demo/

File: src/examples/tensorflow/ssd300_demo/README.md [hunk 2]
```diff
@@ -16,31 +16,23 @@ For this demo, launch one inf1.xlarge EC2 instance. We recommend using the lates
...
-pip install pycocotools==2.0.0
-pip install tensorflow-serving-api==1.15.0 --no-deps
 ```
 
 2. Clone NVIDIA's DeepLearningExamples repo that contains PyTorch SSD300.
```
Comment (aws-mesharma) [inline src/examples/tensorflow/ssd300_demo/README.md:38]: add cd .. after this command

File: src/examples/tensorflow/ssd300_demo/README.md [hunk 3]
```diff
@@ -16,31 +16,23 @@ For this demo, launch one inf1.xlarge EC2 instance. We recommend using the lates
 
 Please configure your ubuntu16/ubuntu18/yum repo following the steps in the [Neuron installation guide](../../../../docs/neuron-install-guide.md) in order to install `tensorflow-model-server-neuron`.
```
Comment (aws-mesharma) [inline src/examples/tensorflow/ssd300_demo/README.md:17]: this is confusing when we have everything in requirements.txt Can we remove this line ?

File: src/examples/tensorflow/ssd300_demo/README.md (hunk 1 above)
Comment (awshaichen) [reply src/examples/tensorflow/ssd300_demo/README.md:35]: They already checked out this repo and are looking at this file. There is no need to check it out again.

File: src/examples/tensorflow/ssd300_demo/README.md (hunk 2 above)
Comment (awshaichen) [reply src/examples/tensorflow/ssd300_demo/README.md:38]: Removed use of subshell to simplify things.

File: src/examples/tensorflow/ssd300_demo/README.md (hunk 3 above)
Comment (awshaichen) [reply src/examples/tensorflow/ssd300_demo/README.md:17]: This is for apt packages.

Comment (aws-mesharma) [reply src/examples/tensorflow/ssd300_demo/README.md:17]: but we have sudo apt install commands in this tutorial as well. User literally only needs to run sudo apt install tensorflow-model-server-neuron

Comment (awshaichen) [reply src/examples/tensorflow/ssd300_demo/README.md:17]: There is only one `sudo apt install tensorflow-model-server-neuron` in this tutorial. It is to make this tutorial self-contained. The configuration step in the installation guide only installs runtime packages.

Comment (aws-mesharma) [reply src/examples/tensorflow/ssd300_demo/README.md:17]: Then, we should remove the line 'to install `tensorflow-model-server-neuron`.

**Review by aws-mesharma — APPROVED** [review]

**Pull request merged**